code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1206,3 +1206,10 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 	ret.color = color
 	ret.appearance_flags = (PIXEL_SCALE) | flags
 	return ret
+
+/proc/emissive_overlay_image(icon,icon_state,color,flags)
+	var/image/ret = image(icon,icon_state)
+	ret.color = color
+	ret.appearance_flags = (PIXEL_SCALE) | flags
+	ret.plane = EMISSIVE_PLANE
+	return ret
```

Comment:
don't we already have this in helpers

Reply:
(neither works)
i'll remove this one though